jedi/evaluate/gradual/conversion.py

```diff
@@ -40,7 +40,10 @@ def _stub_to_python_context_set(stub_context, ignore_compiled=False):
 
 
 def _infer_from_stub(stub_module, qualified_names, ignore_compiled):
-    assert isinstance(stub_module, StubModuleContext), stub_module
+    from jedi import evaluate
+    assert isinstance(stub_module, StubModuleContext) or \
+           isinstance(stub_module, evaluate.compiled.mixed.MixedObject) , stub_module
```

Comment:
Can you just import `from jedi.evaluate.compiled.mixed import MixedObject`? I like it better that way :)

Also please remove the comment below, I will merge once that's done.

Reply:
Me too, however we are back to the circular import, because mixed imports `to_stub` from `conversion.py`.
I saw you plan to merge `to_stub` anyway so we could defer this to a new issue and fix the import this way.

Reply:
Why would it be a circular import? As long as it's in the function it's no problem, it's just if you move it out of the function that it's going to be circular.